Tasks/PowerShellV2/task.json

```diff
@@ -132,6 +132,15 @@
             "defaultValue": "",
             "helpMarkDown": "Working directory where the script is run.",
             "groupName": "advanced"
+        },
+        {
+            "name": "useCallOperator",
+            "type": "boolean",
+            "label": "Use Call operator - &",
+            "required": false,
+            "defaultValue": "false",
+            "helpMarkDown": "This input allows to execute powershell scritpts using '&' operator instead of default '.'.",
```

Comment:
Could we make name and markdown of this input more descriptive? So it will be more clear that with & there won't be ability to use previous context.

Reply:
Sure, I've renamed input and updated markdown text added some info about changing the scope when using the operator.